programs/stake/src/stake_state.rs

```diff
@@ -3292,24 +3120,24 @@ mod tests {
         let to_account = AccountSharedData::new_ref(1, 0, &system_program::id());
         let to_keyed_account = KeyedAccount::new(&to, false, &to_account);
 
-        // Withdrawing account down to only rent-exempt reserve should succeed before feature, and
-        // fail after
+        // Withdrawing account down to minimum balance should succeed
```

Comment:
```suggestion
        // Withdrawing account down to minimum delegation should succeed
```

Reply:
@t-nelson Assuming `MINIMUM_DELEGATION = 1`, this is withdrawing down to `rent_exempt_reserve + MINIMUM_DELEGATION`. That's why I put "balance" here. My understanding is definitely not complete, but wanted to double check; should it be "delegation" instead?